website/docs/06_graphql.md

```diff
@@ -324,6 +324,11 @@ mutation {
 Remember that, in case you want to subscribe to a read model that is restricted to a specific set of roles, you must send the **access token** retrieved upon sign-in. Check ["Authorizing operations"](#authorizing-operations) to know how to do this.
 :::
 
+:::note
+You can disable the creation of the artifacts needed to manage subscriptions by using `config.enableSubscriptions=false`.
```

Comment:
Not sure whether the word `artifact` conveys the desired meaning here (I associate it to packages or dependencies, maybe it is only me)

My suggestion is to change it to something like this:
```suggestion
You can disable the creation of all the infrastructure and functionality needed to manage subscriptions by setting `config.enableSubscriptions=false` in your `Booster.config` block.
```